resources/assets/scripts/base/angular-chart.js

```diff
@@ -0,0 +1,336 @@
+(function (factory) {
```

Comment:
A few versions of this library seem to exist on npm. Use one of those and add it to `package.json` instead of committing in the library itself.